arrow-core-data/src/main/kotlin/arrow/core/Either.kt

```diff
@@ -1077,6 +1082,12 @@ sealed class Either<out A, out B> : EitherOf<A, B> {
         .fold({ t: Throwable -> throwable(t) }, { b: B -> b.right() })
         .fold({ t: Throwable -> unrecoverableState(t); throw t }, { b: B -> b })
   }
+
+  fun <C> mapConst(b: C): Either<A, C> =
+    map { b }
```

Comment:
Should we rename the parameter as `c`?

Reply:
Yes, we should. Should probably also re-check the `Validated.kt` extension functions since some are using `<A, B> and some are using `<E, A>` and others <L, R>.

Reply:
Updated.